python/ray/data/read_api.py

```diff
@@ -933,10 +933,10 @@ def read_json(
     ignore_missing_paths: bool = False,
     **arrow_json_args,
 ) -> Dataset:
-    """Creates a :class:`~ray.data.Dataset` from JSON files.
+    """Creates a :class:`~ray.data.Dataset` from JSON and JSONL files.
 
     Examples:
-        Read a file in remote storage.
+        Read a JSON file in remote storage.
```

Comment:
`logs.json` is actually a JSONL file with the JSON extension. To avoid confusion, maybe we should rename `logs.json` to `logs.jsonl`? We also don't need to keep both examples because they both read JSONL.

Reply:
Yeah good catch. I will add a file with a single line of JSON, to represent JSON file.